<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-VRIV7SIU" failures="1" tests="2" name="testcases.TestLogin" time="3.612" errors="0" timestamp="17 Mar 2022 18:50:30 GMT">
  <testcase classname="testcases.TestLogin" name="enter_SiginDetails" time="0.935"/>
  <testcase classname="testcases.TestLogin" name="create_new_dashboard" time="2.677">
    <failure message="Input must be set" type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: Input must be set
	at org.openqa.selenium.internal.Require.nonNull(Require.java:56)
	at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:97)
	at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:128)
	at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:61)
	at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:48)
	at pages.CreateDashboardPage.<init>(CreateDashboardPage.java:17)
	at testcases.TestLogin.create_new_dashboard(TestLogin.java:57)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
    </failure>
  </testcase> <!-- create_new_dashboard -->
</testsuite> <!-- testcases.TestLogin -->
